lru_cache/lru_cache.py

```diff
@@ -16,8 +24,14 @@ def __init__(self, limit=10):
     Returns the value associated with the key or None if the
     key-value pair doesn't exist in the cache.
     """
+
     def get(self, key):
-        pass
+        if key in self.storage.keys():
```

Comment:
Awesome job. We can remove the `.keys()` here to make the first line be `if key in self.storage`